docs/adr/0003-use-queue-for-backend-services.md

```diff
@@ -0,0 +1,140 @@
+# Use queues for more resilient backend services
+
+- Status: proposed
```

Comment:
Q: is this actually proposed? Sounds like you settled on "GCP Pub/Sub" below.

Reply:
Yes we did and it's being tested on stage, I'll update this section thanks! I'd like to bring it up in the tech meeting tomorrow and just make sure everyone is aligned now that it's working on stage - we should ship to prod soon if all goes well.